### PR TITLE
add suborg admin role to admin on application acceptance

### LINKED ISSUE #453: suborg admin user profile
It seems that suborg admins arent being given the correct user profile when there application is accepted...

### PR DESCRIPTION
**Pull request by diwash007**
# Description
Adds suborg admin role to the admin on application acceptance

Fixes #453 

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
By creating an application and accepting by Admin

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have not added a commit to any .db files as part of my pull request
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by botanicvelious** [comment]
This will need to add the suborg admin role for the main suborg admin as well as the 2 additional emails that can be added.

**Comment by diwash007** [comment]
> This will need to add the suborg admin role for the main suborg admin as well as the 2 additional emails that can be added.

Sure, I will add that.

**Comment by botanicvelious** [comment]
your accept_admin function will need to send reglinks if the admin doesnt exist

File: suborg/views.py [hunk 1]
```diff
@@ -116,7 +117,35 @@ def post_register(request):
 def accept_application(request, application_id):
     if request.method == "GET":
         application = SubOrgDetails.objects.get(id=application_id)
-        application.accept()
+        suborg = SubOrg.objects.filter(suborg_name=application.suborg_name)
```
Comment (botanicvelious) [inline suborg/views.py]: probably better to use .get then to filter

**Pull request merged**